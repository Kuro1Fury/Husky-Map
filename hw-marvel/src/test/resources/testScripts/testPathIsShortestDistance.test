# Test that the path is the shortest based on distance

CreateGraph A

AddNode A n1
AddNode A n2
AddNode A n3
AddEdge A n1 n2 Edge1
AddEdge A n2 n3 Edge3
AddEdge A n1 n3 Edge2

FindPath A n1 n3
