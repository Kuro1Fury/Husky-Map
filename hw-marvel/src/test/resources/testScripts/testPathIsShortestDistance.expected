# Test that the path is the shortest based on distance

created graph A

added node n1 to A
added node n2 to A
added node n3 to A
added edge Edge1 from n1 to n2 in A
added edge Edge3 from n2 to n3 in A
added edge Edge2 from n1 to n3 in A

path from n1 to n3:
n1 to n3 via Edge2
